vertical-pod-autoscaler/e2e/v1/common.go

```diff
@@ -56,6 +56,8 @@ const (
 
 	defaultHamsterReplicas     = int32(3)
 	defaultHamsterBackoffLimit = int32(10)
+	// dummyRecommender for self update vpa status, avoid being modified by the default recommender
+	dummyRecommender = "notExistRecommender"
```

Comment:
Please don't set this as recommender.

We don't need this. When running e2e tests with [run-e2e.sh](https://github.com/kubernetes/autoscaler/blob/114a35961a85efdf3f36859350764e5e2c0c7013/vertical-pod-autoscaler/hack/run-e2e.sh) only some components are deployed:
- For [admission-controller](https://github.com/kubernetes/autoscaler/blob/114a35961a85efdf3f36859350764e5e2c0c7013/vertical-pod-autoscaler/hack/deploy-for-e2e.sh#L47) tests only admission contoller is deployed,
- for [actuation](https://github.com/kubernetes/autoscaler/blob/114a35961a85efdf3f36859350764e5e2c0c7013/vertical-pod-autoscaler/hack/deploy-for-e2e.sh#L53) tests admission-controller and updater are deployed

So there is no need to set alternative recommender inside tests.